tests/test_config.yaml

```diff
@@ -10,10 +10,13 @@ levels:
       - kernel.timer.behavior
       - arch.interrupt
       - boards.*
+      - drivers.gpio.1pin
+      - drivers.console.uart
+      - drivers.entropy
   - name: acceptance
     description: >
       More coverage
+    inherits:
```

Comment:
Could we also describe (with comments) the 'acceptance' scope purpose to make it clear ?
And what is the principle here for levels: now it is a hierarchy, not two different scopes.

Reply:
The idea is that this file will be used during on-target tests and such results will be published to the upstream. IMO the idea is that "acceptance" is a broader scope than "smoke". The "acceptance" scope was already extending the "smoke" one by repeating the phrases. I wanted to increase the scope and found copy-pasting to both scopes redundant and error-prone. 
I don't know if this file is used anywhere, don't think so. From my understanding the idea was to have something to start with as a base for "tier 1 platforms" testing. I also think that there will be several more changes to it. E.g. the "levels" might have more sense to be called "daily", "weekly", "release", etc. when we decide on the frequency and scope of those tests. The same with description. It is already there: "More coverage". I know it doesn't tell much. However, I am reluctant to replace it with sth more verbose now, because as explained, I think the idea for its usage is not finalized and can still evolve.

Reply:
I found that this file is loaded by default by twister https://github.com/zephyrproject-rtos/zephyr/issues/65682. If it must stay like this and there is sth I am missing about this file, I will create a new file for "tier 1 testing"